apps/design/backend/src/app.ts

```diff
@@ -165,6 +178,12 @@ function buildApi({ store }: { store: Store }) {
         }
       }
 
+      // TODO catch-22: we need the hash of the election definition in the QR
+      // code on the ballot, but we also need to lay out the ballot first to get
+      // the gridLayouts to put in the election definition. Likely will need to
+      // do two passes of laying out the ballot - one to generate the
+      // gridLayouts, then one to actually generate the ballots. Then we'll need
+      // to export these at the same time.
```

Comment:
Will address this in an upcoming PR where I add election hash checking to the interpreter.